src/main/java/sdk/sample/Cleanup.java

```diff
@@ -0,0 +1,184 @@
+                        }
+                    }
+                }
+            }
+        }
+
+         /*
+          Clean up volumes
+         */
+        Utils.writeConsoleMessage("Cleaning up Volume(s)...");
+        for (ModelNetAppAccount account : config.getAccounts())
+        {
+            if (account.getCapacityPools() != null)
+            {
+                for (ModelCapacityPool pool : account.getCapacityPools())
+                {
+                    if (pool.getVolumes() != null)
+                    {
+                        for (ModelVolume volume : pool.getVolumes())
+                        {
+                            String[] parameters = {account.getResourceGroup(), account.getName(), pool.getName(), volume.getName()};
+                            VolumeInner volumeInner = (VolumeInner) CommonSdk.getResource(anfClient, parameters, VolumeInner.class);
+                            if (volumeInner != null) {
+                                try {
+                                    anfClient.getVolumes().beginDelete(account.getResourceGroup(), account.getName(), pool.getName(), volume.getName()).getFinalResult();
+
+                                    CommonSdk.waitForNoANFResource(anfClient, volumeInner.id(), VolumeInner.class);
+                                    Utils.writeSuccessMessage("Successfully deleted Volume: " + volumeInner.id());
+                                } catch (Exception e) {
+                                    Utils.writeErrorMessage("An error occurred while deleting Volume: " + volumeInner.id());
+                                    Utils.writeConsoleMessage("Error: " + e);
+                                    throw e;
+                                }
+                            }
+                        }
+                    }
+                    else
+                    {
+                        Utils.writeConsoleMessage("No Volumes defined for Account: " + account.getName() + ", Capacity Pool: " + pool.getName());
+                    }
+                }
+            }
+        }
+
+         /*
+          Clean up capacity pools
+         */
+        Utils.writeConsoleMessage("Cleaning up Capacity Pool(s)...");
+        for (ModelNetAppAccount account : config.getAccounts())
+        {
+            if (account.getCapacityPools() != null)
+            {
+                for (ModelCapacityPool pool : account.getCapacityPools())
+                {
+                    String[] parameters = {account.getResourceGroup(), account.getName(), pool.getName()};
+                    CapacityPoolInner capacityPool = (CapacityPoolInner) CommonSdk.getResource(anfClient, parameters, CapacityPoolInner.class);
+                    if (capacityPool != null)
+                    {
+                        try
+                        {
+                            anfClient.getPools().beginDelete(account.getResourceGroup(), account.getName(), pool.getName()).getFinalResult();
```

Comment:
There has to be a waiting logic after the pool deletion. we have an issue is running currently where it takes few seconds after the pool deletion to be able to delete the account